app/lib/activitypub/activity/create.rb

```diff
@@ -21,9 +21,11 @@ def perform
 
   def process_status
     media_attachments = process_attachments
+    create_status_params = status_params
+    create_status_params['media_attachments'] = media_attachments
 
     ApplicationRecord.transaction do
-      @status = Status.create!(status_params)
+      @status = Status.create!(create_status_params)
```

Comment:
```ruby
@status = Status.new(status_param)

process_tags(@status)
attach_media(@status, media_attachments)

@status.save!
```

🤔

cc @abcang

Reply:
That does not work, I think, because `attach_media` refers to the ID of the status.

Reply:
hm

Reply:
They can be reordered.. You can create media unattached first, and then pass to Status.create like in PostStatusService...

By the way, sorry I missed this..

Reply:
I think that it is good to give media_attachments as a parameter of Status. And `attach_media` will not be used.